### PR TITLE
syst_title is missing in 731

### PR DESCRIPTION
**Pull request by sbcgua**
bug fix: syst_title defined as class type in zcl_abapgit_free_sel_dialog and reused respectively. The rest are resorderings

### CODE REVIEW

File: src/ui/zcl_abapgit_free_sel_dialog.clas.abap [hunk 1]
```diff
@@ -17,9 +17,12 @@ CLASS zcl_abapgit_free_sel_dialog DEFINITION
         text             TYPE rsseltext,
       END OF ty_free_sel_field,
       ty_free_sel_field_tab TYPE STANDARD TABLE OF ty_free_sel_field WITH DEFAULT KEY.
+
+    TYPES: ty_syst_title TYPE sychar70.
```
Comment (mbtools) [inline src/ui/zcl_abapgit_free_sel_dialog.clas.abap:21]: oh oh @larshp won't like this ;-)

`type c length 70`

Comment (sbcgua) [reply src/ui/zcl_abapgit_free_sel_dialog.clas.abap]: controversial. But ok, fixed

Comment (larshp) [reply src/ui/zcl_abapgit_free_sel_dialog.clas.abap]: `syst_title` did not exist in 731, `sychar70` might not exist in ABC, ie. best to avoid both

Comment (larshp) [reply src/ui/zcl_abapgit_free_sel_dialog.clas.abap]: or make it `TYPE sy-title`, if its semantically a sy title

**Comment by larshp** [comment]
suggest adding `syst_title` and `sychar70` in https://github.com/larshp/abapGit/blob/master/abaplint.json#L97 to make sure this error will not occur in the future

**Pull request merged**